cypress/integration/query_builder_test.js

```diff
@@ -0,0 +1,190 @@
+describe("Query builder test", function() {
+    beforeEach(function() {
+      cy.visit("/biotestmine/querybuilder");
+    });
+
+    it("allows you to select data type and summary attributes with model browser", function() {
+      cy.get('.model-browser-root').within(() => {
+        cy.get('select').then($option => {$option.val("Protein")}).parent().trigger('change');
```

Comment:
Can you add a cypress command for this (taking the class you want to select as argument)? It's used many places and has complex syntax.